mnist_test.go

```diff
@@ -18,9 +18,9 @@ type MLPMNISTNet struct {
 
 func NewMNISTNet() *MLPMNISTNet {
 	r := &MLPMNISTNet{
-		FC1: nn.Linear(28*28, 512, false),
-		FC2: nn.Linear(512, 512, false),
-		FC3: nn.Linear(512, 10, false)}
+		FC1: nn.Linear(28*28, 512, true),
```

Comment:
Can you please explain why would this work on random seed initialization?

Reply:
The `Linear` enables the `bias` by default in C++, I think it's just a bug in the previous implementation in Go.
c.f. https://github.com/pytorch/pytorch/blob/v1.6.0/torch/csrc/api/include/torch/nn/options/linear.h#L25